portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase

```diff
@@ -0,0 +1,402 @@
+		}
+
+		task ("Then: Verify No cookies are set, not even the functional cookie: GUEST_LANGUAGE_ID verify cookie banner with site level is present.") {
+			CookieBanner.verifyCookieBannerIsPresent();
+
+			// AssertTextEquals(
+			// 	locator1 = "CookieBannerAdmin#PRIVACY_POLICY_LINK",
+			// 	value1 = SITE_LEVEL_1);
+
+			// CookieBannerAdmin.verifyAllConfigurationElementsAreConfigured(text = "SITE_LEVEL");
+
+			CookieConsentPanel.verifyCookiesAreNotPresent();
+
+			CookieHelper.checkSelectedCookieIsNotPresent(
+				cookieName = "GUEST_LANGUAGE_ID",
+				expectedCookieValue = "en_US");
+		}
+
+		task ("And: End User first visits the instance,Verify cookie banner with instance level is present.") {
+			UserBar.signOut();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http:localhost.com:8080",
+				userEmailAddress = "test@liferay.com");
+
+			CookieBanner.verifyCookieBannerIsPresent();
+
+			// AssertTextEquals(
+			// 	locator1 = "CookieBannerAdmin#PRIVACY_POLICY_LINK",
+			// 	value1 = SITE_LEVEL_2);
+
+			// CookieBannerAdmin.verifyAllConfigurationElementsAreConfigured(text = "INSTANCE_LEVEL");
+		}
+	}
+
+	@description = "This is a use case for LPS-170904 TC-1. Verify no cookies are set in Site level and Explicit mode, when the user is a first-time visitor"
+	@priority = 5
+	test VerifyNoCookiesForFirstVisitInSiteLevelExplicitMode {
+		property test.name.skip.portal.instance = "CookiesSiteLevelUseCase#VerifyNoCookiesForFirstVisitInSiteLevelExplicitMode";
+
+		task ("Given: Site Administrator enabled the cookie preference handling in site level and the explicit cookie consent mode.") {
+			for (var count : list "1,2") {
+				JSONUser.addUser(
+					userEmailAddress = "userea${count}@liferay.com",
+					userFirstName = "userfn${count}",
+					userLastName = "userln${count}",
+					userScreenName = "usersn${count}");
+
+				JSONUser.setFirstPassword(
+					agreeToTermsAndAnswerReminderQuery = "true",
+					requireReset = "false",
+					userEmailAddress = "userea${count}@liferay.com");
+
+				JSONUser.addUserToSite(
+					groupName = "Guest",
+					userEmailAddress = "userea${count}@liferay.com");
+			}
+
+			JSONRole.assignSiteRoleToUser(
+				groupName = "Guest",
+				roleTitle = "Site Administrator",
+				userEmailAddress = "userea1@liferay.com");
+
+			Site.openSiteSettingsAdmin(siteURLKey = "guest");
+
+			Site.addVirtualHostsURLCP(
+				pageVirtualHosts = "true",
+				pageVirtualHostURL = "www.able.com");
+
+			UserBar.signOut();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "userea1@liferay.com");
+
+			CookiePreferenceHandlingAdmin.enablePreferenceHandlingWithCookieConsentInSiteSettings(
+				baseURL = "http://www.able.com:8080",
+				portlet = "Preference Handling",
+				siteURLKey = "guest");
+		}
+
+		task ("When: End User first visits the site") {
+			UserBar.signOut();
+
+			User.firstLoginUI(
```

Comment:
I think 2 user and first login by the second user is not needed, we could simulate a private window by 
Deleting all existing cookies by CookieHelper.deleteAllCookiesAndRefreshPage(); macro
And asserting the TC's expectation.
What do you think?
cc: @JoyceWang08211

Reply:
done

Reply:
I think we could login the site host(aka http://www.able.com:8080) with instance admin(aka test test) and enable explicit cookie consent mode. Then first login site host with the new user. That means adding only one new user is good enough.

Reply:
done